src/app/wallet/view.nim

```diff
@@ -257,7 +264,12 @@ QtObject:
   proc toggleAsset*(self: WalletView, symbol: string, checked: bool, address: string, name: string, decimals: int, color: string) {.slot.} =
     self.status.wallet.toggleAsset(symbol, checked, address, name, decimals, color)
     for account in self.status.wallet.accounts:
-      self.accounts.updateAssetsInList(account.address, account.assetList)
+      if account.address == self.currentAccount.address:
+        self.currentAccount.setAccountItem(account)
+      else: 
+        self.accounts.updateAssetsInList(account.address, account.assetList)
+    self.accountListChanged()
+    self.currentAccountChanged()
```

Comment:
Don't wanna make things more complicated than necessary, but it would be probably cool to have this change in a separate `fix(wallet): ..` commit. No big deal if you don't feel like pulling this out.

Reply:
I would do it, but cascading this change so that this branch has this fix in it would make things a fair bit complicated as this is already the base branch for other braches/PRs